tests/framework/extensions/clusters/bundledclusters/list.go

```diff
@@ -14,9 +14,16 @@ import (
 func (bc *BundledCluster) ListAvailableVersions(client *rancher.Client) (versions []string, err error) {
 	switch bc.Meta.Provider {
 	case clusters.KubernetesProviderRKE:
-		versions, err = available.ListRKE1AvailableVersions(client, bc.V3)
-		if err != nil {
-			return
+		if bc.Meta.IsImported {
+			versions, err = available.ListRKE1ImportedAvailableVersions(client, bc.V3)
+			if err != nil {
+				return
+			}
+		} else if !bc.Meta.IsImported {
```

Comment:
nit, I don't think the else if or an else is even necessary,

Reply:
These `else if `s  were before the imported and custom separation. I was unsure if we would need more separation for them. Definitely redundant, addressed. Also updated the other list items in this file